Mathlib/Topology/Maps/Basic.lean

```diff
@@ -136,6 +136,7 @@ lemma continuousAt_iff' (hf : IsInducing f) {x : X} (h : range f ∈ 𝓝 (f x))
     ContinuousAt (g ∘ f) x ↔ ContinuousAt g (f x) := by
   simp_rw [ContinuousAt, Filter.Tendsto, ← hf.map_nhds_of_mem _ h, Filter.map_map, comp]
 
+@[fun_prop]
```

Comment:
This, and the `Embedding` and `OpenEmbedding` ones below, is a recipe for disaster since those structures are not marked `fun_prop` themselves. So the only way these lemmas would succeed with `fun_prop` is if there were a discharger that supplied these hypotheses (which seems rare, since that's what `fun_prop` is supposed to do).

Now, maybe we *should* mark those as `fun_prop`, but that's another discussion.

Reply:
Good point, thanks for being careful! I have reverted these changes for now.